experimental-examples/kitchen-sink/tina/custom-component-reference-select/reference-field-schema.tsx

```diff
@@ -49,6 +26,34 @@ const referenceField = {
           return s.path
       }
     },
+    //Static example - user can define which field and value they want to filter
+    // collectionFilter: {
+    //   author: {
+    //     location: 'melbourne',
+    //   },
+    //   post: {
+    //     title: 'hello world',
+    //   },
+    // },
+    //Dynamic example - user can define the function they want here but need to make sure the return type match the schema
+    collectionFilter: () => {
+      return {
+        author: {
+          location: (hostname: string): LocationEnum => {
+            switch (true) {
+              case hostname.includes('ndcmelbourne.com'):
```

Comment:
**question**: Are we able to use example data consistent with https://www.ssw.com.au/rules/use-generic-consistent-names-on-examples/ ?

Reply:
Done - fix to use bob-northwind